fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -346,15 +346,16 @@ export default Ember.Controller.extend({
     },
 
     editGlobalServiceConfig: function() {
-      var planParams = [];
+      var planParams = Ember.A();
       this.get('model.plan.parameters').forEach(function(param) {
-        if (param.get('id').indexOf('::') === -1) {
+        // only show parameters without :: in the name
+        if (param.get('id').indexOf('::') !== -1) {
```

Comment:
thanks @tzumainn, the issue was that my API mock was only partial. It's fixed now and all is working